mappings/SkipButtonState.mapping

```diff
@@ -1 +1,15 @@
 CLASS rbm SkipButtonState
+	FIELD a skipAdRenderer Lalhc;
+	FIELD b contentMetadata Lrbr;
+	FIELD c adCountMetadata Lqty;
+	FIELD d skipState I
+	FIELD e hidden Z
+	FIELD f timeRemainingUntilSkippableMillis I
+	FIELD g timeRemainingInAdMillis I
+	FIELD h breakType Lqxl;
+	FIELD i DRCtaEnabled Z
```

Comment:
Do we know what `DRCta` means?

Reply:
Can you replace all `Cta` occurrences with `CallToAction`? I know, this makes rather long names, but I don't think `Cta` is such a common abbreviation, and our mappings' main purpose to make names more readable anyway, not short